examples/pets/src/2_main.tsx

```diff
@@ -24,14 +25,19 @@ import { PetAccount, PetPost } from "./1_schema.ts";
 
 const appName = "Jazz Rate My Pet Example";
 
-const auth = PasskeyAuth<PetAccount>({
+const passkeyAuth = PasskeyAuth<PetAccount>({
     appName,
     Component: PrettyAuthUI,
     accountSchema: PetAccount,
 });
 
+const authFallback = DemoAuth<PetAccount>({
+    appName,
+    accountSchema: PetAccount,
+});
+
 const Jazz = createJazzReactContext({
-    auth,
+    auth: supportsWebAuthn ? passkeyAuth : authFallback,
```

Comment:
@aeplay 

Tried to emulate WebAuthN using the Chrome virtual authenticator but didn't work:

```
  await devTools.send('WebAuthn.enable');
  await devTools.send('WebAuthn.addVirtualAuthenticator', {
    options: {
      protocol: 'ctap2',
      transport: 'internal',
      hasUserVerification: true,
      isUserVerified: true,
      hasResidentKey: true,
    },
  });
```

So I've decided to go with a fallback auth when the passkeys aren't supported.

Does it look good for you?

Reply:
Yes that's great, eventually we probably want to bake a fallback into PasskeyAuth itself